src/cloudinary-video.component.spec.ts

```diff
@@ -74,7 +74,7 @@ describe('CloudinaryVideo', () => {
   describe('videos with nested transformations', () => {
     @Component({
       template: `<cl-video id="video1" public-id="sample_video">
-            <cl-transformation width="300" crop="scale" overlay="text:roboto_35_bold:SDK"></cl-transformation>
+            <cl-transformation width="300" crop="scale" keyframe_interval="12" overlay="text:roboto_35_bold:SDK"></cl-transformation>
```

Comment:
This test is using only `snake_case`.
Please add a test that shows that `kebab-case` is also supported, i.e. `keyframe-interval="12"`.